### DISCUSSION TITLE
Compatibility with Nodejs v22

### DISCUSSION THREAD

**Discussion by jashan777** [remarkjs/remark#1334] [Q&A]
will the below packages , will be compatible with Nodejs v22
"rehype-katex": "^5.0.0",
"remark-gfm": "^3.0.1",
"remark-math": "^5.1.1",

**Comment by remcohaszing** [comment]
I suggest updating to the latest version. But why wouldn’t they be compatible?

**Reply by jashan777** [reply]
actually when i was reading the documentation for rehype-katex 
<img width="759" alt="image" src="https://github.com/remarkjs/remark/assets/49744054/a025861e-efce-4e7a-83e2-80e1c55d609a">
and this led me to think if using latest version of node might cause some problem.

**Reply by wooorm** [reply]
“compatible with maintained versions of Node”

**Reply by remcohaszing** [reply]
Node.js updates usually don’t break JavaScript APIs. In my experience only native bindings ever caused backwards compatibility issues. Since all of unified uses JavaScript (I think), I wouldn’t worry too much about compatibility with current and future versions of Node.js.

**Reply by jashan777** [reply]
I think the example "rehype-katex@^7, compatible with Node.js 16." in the second paragraph can be improved, as it feels like there is a cap on Node.js versions.
btw thank you guys for replying, it means a lot.

**Reply by wooorm** [reply]
That paragraph is about the lowest (unmaintained) version of Node. The paragraph before is about the highest (maintained) version of Node.
